msteams-platform/bots/how-to/conversations/subscribe-to-conversation-events.md

```diff
@@ -1372,6 +1372,24 @@ turnContext, CancellationToken cancellationToken) {
 ```
 * * *
 
+## Uninstall event
+
+> [!NOTE]
+> Uninstall event is currently available only in developer preview.
+
+When you uninstall an app, the bot is also uninstalled. When a user sends a message to your app, they receive a 403 response code. Your bot receives a 403 response code for new messages posted by your bot. The 403 response code has fields that give the reason behind the 403 response, which is either the app was uninstalled or the bot was blocked. After the post uninstall behavior for bots in the personal scope with the Teams and groupChat scopes are aligned, you cannot send or receive messages after an app has been uninstalled.
```

Comment:
The following sentence is not clear, please rephrase:


After the post uninstall behavior for bots in the personal scope with the Teams and groupChat scopes are aligned, you cannot send or receive messages after an app has been uninstalled.